### ISSUE TITLE
Running old test cases / commands (old, deprecated?)

### ISSUE THREAD

**Issue by Zaanmedia** [SeleniumHQ/selenium-ide#65]
Looks like with the new selenium IDE there are less or limited commands you can use/select.

**Example:** Failed: Unknown command: waitForElementPresent

Are those commands on purpose gone? would be nice to have a suggestion to what to use instead in the log if commands needs to be replaced with something else? 

Another issue i'm facing right now is the following:

**Blocked a frame with origin "URL" from accessing a cross-origin frame.**

Im fairly new to IDE and just starting with automation testing! :D

**Comment by schunt1** [comment]
Eeek, that one in particular would be a problem for us; we use it all the time.

Simon

**Comment by corevo** [comment]
If you could upload the test cases where you use these commands that would be great.  
  
In any case, the IDE waits for elements to appear as is, it waits for the page to finish loading, and finish any active network requests,  
if you specifically enter a delay so that an element would appear, then try removing it, and see if it works.  
  
Other than that, what caused **"Blocked a frame with origin "URL" from accessing a cross-origin frame."**?  
Can you send me the command?, are you trying to access an iframe inside the webpage with a different origin  
```html
<iframe src="https://example.com">
  <input id="under-test" />
</iframe>
```
For example trying to access `under-test`, within the origin `https://example.com`, when the current url is not `https://example.com`?

**Comment by Zaanmedia** [comment]
Another thing i noticed i used **comments** that may could break my the tests? 

`<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head profile="http://selenium-ide.openqa.org/profiles/test-case">
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="selenium.base" href="https://qa1.thenextad.com/" />
<title>OpenTargeting-Search-type-enter</title>
</head>
<body>
<table cellpadding="1" cellspacing="1" border="1">
<thead>
<tr><td rowspan="1" colspan="3">[Create] Open Targeting audience</td></tr>
</thead><tbody>

<tr>
	<td>open</td>
	<td>/audiences</td>
	<td></td>
</tr>
<tr>
	<td>waitForPageToLoad</td>
	<td></td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=button:contains(&quot;Create new audience&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=button:contains(&quot;Create new audience&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=.btn-group.open &gt; ul.dropdown-menu</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=a:contains(&quot;Open targeting&quot;):nth(1)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>link=General</td>
	<td></td>
</tr>
<tr>
	<td>type</td>
	<td>name=name</td>
	<td>Open Targeting audience ${timeStampEU}</td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=input[type=&quot;search&quot;]</td>
	<td></td>
</tr>
<tr>
	<td>sendKeys</td>
	<td>css=input[type=&quot;search&quot;]</td>
	<td>Dutch</td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=li:contains(&quot;Dutch&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>clickAt</td>
	<td>css=li:contains(&quot;Dutch (Belgium)&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>verifyElementPresent</td>
	<td>css=li:contains(&quot;Dutch (Belgium)&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>sendKeys</td>
	<td>css=input[type=&quot;search&quot;]</td>
	<td>German</td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=li:contains(&quot;German&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>clickAt</td>
	<td>css=li:contains(&quot;German&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>verifyElementPresent</td>
	<td>css=li:contains(&quot;German&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>type</td>
	<td>name=min-age</td>
	<td>18</td>
</tr>
<tr>
	<td>type</td>
	<td>name=max-age</td>
	<td>45</td>
</tr>
<tr>
	<td>select</td>
	<td>name=genders</td>
	<td>label=Male</td>
</tr>
<tr>
	<td>verifySelectedValue</td>
	<td>name=genders</td>
	<td>1</td>
</tr>

<tr>
	<td>click</td>
	<td>link=Locations</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>name=location-types</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>name=location-types value=recent</td>
	<td>recent</td>
</tr>
<tr>
	<td>verifyChecked</td>
	<td>name=location-types value=recent</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>name=location-types value=home</td>
	<td>home</td>
</tr>
<tr>
	<td>verifyChecked</td>
	<td>name=location-types value=home</td>
	<td></td>
</tr>


<tr>
	<td>waitForElementPresent</td>
	<td>css=.select2-selection:contains(&quot;Add a country, state/province, city or ZIP&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>clickAt</td>
	<td>css=.select2-selection:contains(&quot;Add a country, state/province, city or ZIP&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=input[type=&quot;search&quot;]</td>
	<td></td>
</tr>
<tr>
	<td>sendKeys</td>
	<td>css=input[type=&quot;search&quot;]</td>
	<td>Netherlands</td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=li:contains(&quot;Netherlands&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>clickAt</td>
	<td>css=li:contains(&quot;Netherlands&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>verifyElementPresent</td>
	<td>css=.selection:contains(&quot;Netherlands&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=button:contains(&quot;Add location&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=button:contains(&quot;Add location&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=.include:contains(&quot;Included&quot;):nth(1)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=.include:contains(&quot;Included&quot;):nth(1)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=.select2-selection:contains(&quot;Add a country, state/province, city or ZIP&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>clickAt</td>
	<td>css=.select2-selection:contains(&quot;Add a country, state/province, city or ZIP&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=input[type=&quot;search&quot;]</td>
	<td></td>
</tr>
<tr>
	<td>sendKeys</td>
	<td>css=input[type=&quot;search&quot;]</td>
	<td>Amsterdam</td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=li:contains(&quot;Amsterdam&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>clickAt</td>
	<td>css=li:contains(&quot;Amsterdam, Montana, United States&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>verifyElementPresent</td>
	<td>css=.selection:contains(&quot;Amsterdam, Montana, United States&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>verifyElementPresent</td>
	<td>name=radius</td>
	<td></td>
</tr>
<tr>
	<td>select</td>
	<td>css=select[name=&quot;radius&quot;]:nth(1)</td>
	<td>label=40 km</td>
</tr>
<tr>
	<td>verifySelectedValue</td>
	<td>css=select[name=&quot;radius&quot;]:nth(1)</td>
	<td>40</td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=button:contains(&quot;Add location&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=button:contains(&quot;Add location&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=button[data-title='Choose custom location']:nth(2)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=button[data-title='Choose custom location']:nth(2)</td>
	<td></td>
</tr>
<tr>
	<td>setSpeed</td>
	<td>250</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=button:contains(&quot;Use location&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=button:contains(&quot;Use location&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=.location-selection:contains('latitude 52.370633')</td>
	<td></td>
</tr>
<tr>
	<td>verifyElementPresent</td>
	<td>css=.location-selection:contains('latitude 52.370633')</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=.location-selection:contains('longitude 4.928472499999998')</td>
	<td></td>
</tr>
<tr>
	<td>verifyElementPresent</td>
	<td>css=.location-selection:contains('longitude 4.928472499999998')</td>
	<td></td>
</tr>
<tr>
	<td>setSpeed</td>
	<td>250</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=input[name='custom_radius']:nth(2)</td>
	<td></td>
</tr>
<tr>
	<td>type</td>
	<td>css=input[name='custom_radius']:nth(2)</td>
	<td>37</td>
</tr>
<tr>
	<td>fireEvent</td>
	<td>css=input[name='custom_radius']:nth(2)</td>
	<td>blur</td>
</tr>
<tr>
	<td>verifyValue</td>
	<td>css=input[name='custom_radius']:nth(2)</td>
	<td>37</td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=button[data-title='Remove location']:nth(2)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=button[data-title='Remove location']:nth(2)</td>
	<td></td>
</tr>

<tr>
	<td>click</td>
	<td>link=Interests &amp; behaviour</td>
	<td></td>
</tr>


<tr>
	<td>waitForElementPresent</td>
	<td>css=i.mdi.mdi-playlist-plus:nth(0)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=i.mdi.mdi-playlist-plus:nth(0)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=span:contains(&quot;Advanced demographics&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span:contains(&quot;Advanced demographics&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=span:contains(&quot;Politics&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span:contains(&quot;Politics&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=span:contains(&quot;US politics (liberal)&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span:contains(&quot;US politics (liberal)&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>verifyElementPresent</td>
	<td>css=span.selection:contains(&quot;US politics (liberal)&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=button:contains(&quot;Add targeting&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=button:contains(&quot;Add targeting&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=i.mdi.mdi-playlist-plus:nth(1)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=i.mdi.mdi-playlist-plus:nth(1)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=span:contains(&quot;Interests&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span:contains(&quot;Interests&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=span:contains(&quot;Entertainment&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span:contains(&quot;Entertainment&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=span:contains(&quot;Games&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span:contains(&quot;Games&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=span.path-item:contains(&quot;Entertainment&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span.path-item:contains(&quot;Entertainment&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=span:contains(&quot;Games&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span:contains(&quot;Games&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=span:contains(&quot;First-person shooter games&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span:contains(&quot;First-person shooter games&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>verifyElementPresent</td>
	<td>css=span.selection:contains(&quot;First-person shooter games&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=button:contains(&quot;Add targeting&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=button:contains(&quot;Add targeting&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=i.mdi.mdi-magnify:nth(3)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=i.mdi.mdi-magnify:nth(3)</td>
	<td></td>
</tr>

<tr>
	<td>sendKeys</td>
	<td>name=global_search</td>
	<td>playstation</td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=li:contains(&quot;PlayStation 4&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=li:contains(&quot;PlayStation 4&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>verifyElementPresent</td>
	<td>css=span.selection:contains(&quot;PlayStation 4&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=button[data-action='remove-or-field']:nth(2)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=button[data-action='remove-or-field']:nth(2)</td>
	<td></td>
</tr>


<tr>
	<td>waitForElementPresent</td>
	<td>css=button:contains(&quot;Narrow audience&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=button:contains(&quot;Narrow audience&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=button:contains(&quot;Narrow audience&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=button:contains(&quot;Narrow audience&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=button:contains(&quot;Add targeting&quot;):nth(2)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=button:contains(&quot;Add targeting&quot;):nth(2)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=i.mdi.mdi-playlist-plus:nth(2)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=i.mdi.mdi-playlist-plus:nth(2)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=span:contains(&quot;Behaviors&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span:contains(&quot;Behaviors&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=span:contains(&quot;Travel&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span:contains(&quot;Travel&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=span:contains(&quot;Frequent Travelers&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span:contains(&quot;Frequent Travelers&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>verifyElementPresent</td>
	<td>css=span.selection:contains(&quot;Frequent Travelers&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=button[data-action='remove-and-field']:nth(1)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=button[data-action='remove-and-field']:nth(1)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=button:contains(&quot;Exclude people&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=button:contains(&quot;Exclude people&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=i.mdi.mdi-playlist-plus:nth(3)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=i.mdi.mdi-playlist-plus:nth(3)</td>
	<td></td>
</tr>

<tr>
	<td>waitForElementPresent</td>
	<td>css=span:contains(&quot;Education and workplace&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span:contains(&quot;Education and workplace&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=span:contains(&quot;Education statuses&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span:contains(&quot;Education statuses&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>waitForElementPresent</td>
	<td>css=span:contains(&quot;Master degree&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>click</td>
	<td>css=span:contains(&quot;Master degree&quot;)</td>
	<td></td>
</tr>
<tr>
	<td>verifyElementPresent</td>
	<td>css=span.selection:contains(&quot;Master degree&quot;)</td>
	<td></td>
</tr>

<tr>
	<td>click</td>
	<td>css=button[type=&quot;submit&quot;]</td>
	<td></td>
</tr>

</tbody></table>
</body>
</html>
`

**Comment by corevo** [comment]
I have added errors to let you guys know which waits can be removed.  
The `type` and `sendKeys` are issues that originate from Selenium upstream (they appeared now because of different bundling technique that the IDE uses), and currently waiting for [them](https://github.com/SeleniumHQ/selenium/pull/5229) to be merged.  
And I still don't know how to replicate "Blocked a frame with origin "URL" from accessing a cross-origin frame.", it seems page specific, and I need the page source to investigate this further.

**Comment by corevo** [comment]
I didn't see you edited your comment,  
The new IDE, uses the "context" of the browser, rather than running in a super user environment.  
The old IDE on the other hand used a stronger context to run itself, which is why it can no longer work, as security measurements, certain things that the old IDE have done, may not be done anymore.  
In case of iframes, you can read [this](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy), and inform your team about how to gain access from the outside into an iframe.  
Basically the iframe and the page have to have the same origin.

**Comment by Zaanmedia** [comment]
What i did now is to understand why things are failing and i need propably need to learn or read something more about it! 

**Just simple thinking from my side:**
When i use the recorder and try to partly mimic the old test case it simply works and records the ugly xpath just what the old IDE did and it runs successfully. :D with ugly i mean its hard to figure out were thing where going wrong and that was the main reason we changed it to css selectors to make it more readable and easy to find things for our selves. So for just a starter i just don't understand our web application is the issue here since recording and running that with the new IDE just works fine? 

So my conclusion would be that those iframes can't be the cause of failure? Or at least that wouldn't be logic?

**Comment by corevo** [comment]
Here is what I believe is the issue,  
You are running the tests against a local version of the app, `http://localhost/`, while the iframes work on a test or production version, e.g. `http://prod.app.com`.  
Since they are not from the same origin, new browser policies prevent you from accessing the contents of that iframe for obvious reasons.  
Accessing the content is a serious security breach, for example, I could place a bank's iframe inside my website, access it, and have the credentials of other users to their bank.  
Thus accessing iframes from different origins is strictly forbidden according to [this policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy)

**Comment by Zaanmedia** [comment]
**New test case:**
`{
	"id": "20c66ccd-9b9e-4d0f-94b9-090283c8b312",
	"name": "Untitled Project",
	"url": "URL",
	"tests": [{
		"id": "c8d2d599-d044-4834-98db-040bfb9504d3",
		"name": "Untitled",
		"commands": [{
			"id": "e16963c8-18c0-45df-a2ce-11bd87650c2d",
			"command": "open",
			"target": "/campaigns-overview",
			"value": ""
		},
		{
			"id": "c70e7a13-801b-447b-82bb-35fa1ee2f251",
			"command": "clickAt",
			"target": "//nav[@id='sidenavbar']/ul/li[5]/a/i",
			"value": "-7,-4"
		},
		{
			"id": "01a47797-668e-4262-ac9f-29de4489e411",
			"command": "clickAt",
			"target": "//nav[@id='sidenavbar']/ul/li[5]/div/ul/li[2]/a/div/p",
			"value": "6,20"
		},
		{
			"id": "d1fd8fc5-bf90-4e45-87ee-d4074763fa98",
			"command": "clickAt",
			"target": "//section[@id='dashboard']/section/header/div/div[2]/button",
			"value": "129,8"
		},
		{
			"id": "987b00c0-4f21-46b4-9d94-53a42e718099",
			"command": "clickAt",
			"target": "xpath=(//a[contains(text(),'Open targeting')])[2]",
			"value": "86,17"
		},
		{
			"id": "32bcecb7-8c19-49d2-92d7-16bda4d3216a",
			"command": "mouseOver",
			"target": "xpath=(//a[contains(text(),'Open targeting')])[2]",
			"value": ""
		},
		{
			"id": "d704e447-ff1a-4d1b-80d4-6fc6b84311ea",
			"command": "mouseOut",
			"target": "xpath=(//a[contains(text(),'Open targeting')])[2]",
			"value": ""
		},
		{
			"id": "2d0790b0-f1c2-45f3-abc5-f4345ebeb8f3",
			"command": "clickAt",
			"target": "name=name",
			"value": "-299,28"
		},
		{
			"id": "28bfa1c6-bd27-4140-bb87-595be48bce6b",
			"command": "type",
			"target": "name=name",
			"value": "NewIDE"
		},
		{
			"id": "e531dde0-11f0-4340-ae3c-78859335d1f1",
			"command": "clickAt",
			"target": "//input[@type='search']",
			"value": "-419,10"
		},
		{
			"id": "a75288ca-2e25-4282-9744-e11c66c4461a",
			"command": "type",
			"target": "//input[@type='search']",
			"value": "Dutch"
		},
		{
			"id": "b2f413aa-68bb-4b5e-b131-709055bffe18",
			"command": "clickAt",
			"target": "name=min-age",
			"value": "-425,7"
		},
		{
			"id": "e15873b7-b444-4001-86df-c652d3009cad",
			"command": "type",
			"target": "name=min-age",
			"value": "18"
		},
		{
			"id": "688c8136-2949-4cf4-a30a-a953b897b7e2",
			"command": "type",
			"target": "name=max-age",
			"value": "45"
		},
		{
			"id": "f2464b39-66d9-4f32-9750-5f42df6e296c",
			"command": "select",
			"target": "name=genders",
			"value": "label=Male"
		},
		{
			"id": "ea60cc0e-1dc5-4cf1-8067-f7aec6490cd6",
			"command": "clickAt",
			"target": "link=Locations",
			"value": "-390,25"
		},
		{
			"id": "bd7c9e82-22e6-4e7c-a6d0-52626252239a",
			"command": "clickAt",
			"target": "//span[@id='react-select-2--value']/div",
			"value": "-340,23"
		},
		{
			"id": "6e381fef-26fd-4c84-9500-d040556f3a43",
			"command": "clickAt",
			"target": "//button[@type='submit']",
			"value": "-424,29"
		},
		{
			"id": "e99d6800-945b-40f1-a255-594c7e4fffd6",
			"command": "runScript",
			"target": ["window.scrollTo(0,0)"],
			"value": ""
		}]
	}],
	"suites": [],
	"urls": ["URL"]
}`

**Log:**

> Running 'Untitled'
1.Trying to execute open on /campaigns-overview...
2.Trying to execute clickAt on //nav[@id='sidenavbar']/ul/li[5]/a/i with value -7,-4...
3.Trying to execute clickAt on //nav[@id='sidenavbar']/ul/li[5]/div/ul/li[2]/a/div/p with value 6,20...
4.Trying to execute clickAt on //section[@id='dashboard']/section/header/div/div[2]/button with value 129,8...
5.Trying to execute clickAt on xpath=(//a[contains(text(),'Open targeting')])[2] with value 86,17...
6.Trying to execute mouseOver on xpath=(//a[contains(text(),'Open targeting')])[2]...
7.Trying to execute mouseOut on xpath=(//a[contains(text(),'Open targeting')])[2]...
8.Trying to execute clickAt on name=name with value -299,28...
9.Trying to execute type on name=name with value NewIDE...
10.Trying to execute clickAt on //input[@type='search'] with value -419,10...
11.Trying to execute type on //input[@type='search'] with value Dutch...
12.Trying to execute clickAt on name=min-age with value -425,7...
13.Trying to execute type on name=min-age with value 18...
Cannot set the selection end
14.Trying to execute type on name=max-age with value 45...
Cannot set the selection end
15.Trying to execute select on name=genders with value label=Male...
16.Trying to execute clickAt on link=Locations with value -390,25...
17.Trying to execute clickAt on //span[@id='react-select-2--value']/div with value -340,23...
18.Trying to execute clickAt on //button[@type='submit'] with value -424,29...
19.Trying to execute runScript on window.scrollTo(0,0)...
'Untitled' completed with 1 error(s)

**Comment by Zaanmedia** [comment]
I runned the old test with removing the iframes. This seems to work but still all the errors throwing me that all those elements are not found.

**Comment by corevo** [comment]
We use the same selector builders as the old one,  
I can only assume that once you remove the iframes the css changes, thus making them not be found

**Comment by Zaanmedia** [comment]
Hmmm this is not the case? 

The source is still completely the same of our application nothing changed. 

The old IDE still plays the test case without any problems.  
The new IDE throws for every single line a error nothing is found.

We removed the iframes **Hotjar** and our **Intercom**.

**Comment by corevo** [comment]
You are correct, it also seems to be a problem originating from selenium, I'll add it to the [diff](https://github.com/SeleniumHQ/selenium/pull/5229)

**Comment by Zaanmedia** [comment]
Awesome! Thanks 👍

**Comment by olds97lss** [comment]
Also appears than an issue with the "open" command. I have tons of scripts and I don't use the base url at all. Instead, the first command I run is to open a website/url. Appears the released version of the new ide doesn't know what to do with that and gets an error. Unless I manually go to the url outside selenium, then run the script. Then it seems to open and run ok.

**Comment by pgroot91** [comment]
@olds97lss The same case for using variables! :|

**Comment by corevo** [comment]
This issue is starting to get a bit off topic.  
@olds97lss If I understood you correctly, you want a way to specify the entire URL in the open command, rather than the combination of Base URL + open?  
@pgroot91 can you explain what kind of behavior you would like variables to have? a global place to declare variables to be used throughout all the test cases?

**Comment by olds97lss** [comment]
@corevo Yes sir. I never really used the base url in my scripts since I tend to write them to hop from one site to another for work. Within one script, I'll open a jsp that fires a query with parameters I add on to the url for updating database items, then I open the same jsp again but with new parameters to do a lookup. After confirming the update was successful, I open the site I need to test, which is a totally different url.

That's just one instance, but I do open full url's directly pretty much all the time since it's easier for me to maintain.

**Comment by olds97lss** [comment]
@corevo I can open my own ticket, just wasn't sure what to do this early on.

**Comment by corevo** [comment]
It's always best to open a ticket if the title doesn't directly reflect your issue, cause if the conversation gets out of hand it becomes hard for me to keep track of it.

Cheers,
Tom

**Comment by drexlma** [comment]
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head profile="http://selenium-ide.openqa.org/profiles/test-case">
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="selenium.base" href="http://www.jobs-m%C3%BCnchen.com/" />
<title>planko</title>
</head>
<body>
<table cellpadding="1" cellspacing="1" border="1">
<thead>
<tr><td rowspan="1" colspan="3">planko</td></tr>
</thead><tbody>
<tr>
	<td>open</td>
	<td>/jobs/?ref=3&amp;ausbildung=show&amp;nebenjob=show&amp;praktikum=show&amp;action=Filter</td>
	<td></td>
</tr>
<tr>
	<td>pause</td>
	<td>3000</td>
	<td></td>
</tr>
<tr>
	<td>clickAndWait</td>
	<td>css=img.logo</td>
	<td></td>
</tr>
<tr>
	<td>clickAndWait</td>
	<td>link=Vollzeit-Jobs</td>
	<td></td>
</tr>
<tr>
	<td>clickAndWait</td>
	<td>css=a.button &gt; span</td>
	<td></td>
</tr>
<tr>
	<td>clickAndWait</td>
	<td>link=JETZT INSERIEREN</td>
	<td></td>
</tr>
</tbody></table>
</body>
</html>




Trying to execute clickAndWait on css=img.logo... Failed:
Unknown command: clickAndWait
5.Trying to execute clickAndWait on link=Vollzeit-Jobs... Failed:
Unknown command: clickAndWait
6.Trying to execute clickAndWait on css=a.button > span... Failed:
Unknown command: clickAndWait

**Comment by corevo** [comment]
Page source?

**Comment by drexlma** [comment]
jobs-regensburg.com

**Comment by corevo** [comment]
Since the new IDE waits automatically for you, all you have to do is change the deprecated `clickAndWait` to `click at`
<img width="419" alt="screen shot 2018-02-21 at 16 36 55" src="https://user-images.githubusercontent.com/888886/36485742-7aef5218-1725-11e8-8ae7-8955cb395267.png">

**Comment by setiseta** [comment]
we use some waitForVisible to check if a site is loaded correctly. not for clicking or something else, just to check if this target is available or visible on page.
any suggestions how to do this with the new IDE?

**Comment by Nachikethan1** [comment]
Why clickandWait command is depreciated from Selenium IDE? I came to know the Wait is done automatically now, but it is not the case here.. i have a page with three buttons, clicking any one will reload the page, i have to record clicking all three buttons but the new version IDE doesn't do the job where it was doing perfectly fine with clickandWait command. I always get two buttons clicked skipping any one with the new version. I tried using Clickat command with no use. Please can you add the ClickandWait command back, i had to use another automation software which provided Clickandwait command to solve it.

**Comment by tourdedave** [comment]
@Nachikethan1 and @setiseta we have added some wait for commands back, which will be in the next release. See https://github.com/SeleniumHQ/selenium-ide/issues/157 for details.

**Comment by 98gmarquee** [comment]
I know this is relatively old, but wasn't sure if it warranted a new ticket being made. I noticed when I import my older IDE scripts, my "waitforvisible" commands are being omitted entirely from the script in the new ide. With some commands, it loads them, but they are invalid and it fails, so then I change them as needed. But the waitforvisible entire line is being omitted.

I noticed this a while back and forgot about it until today when I was importing an older script that had a lot of waitforvisibles in it.

In the instance I'm using it, it's with a shopping cart. I have something similar to the following:
click|id=EE_cart
waitforvisible|xpath for cart header
waitforvisible|xpath for item I just placed in the cart
click|id=EE_cart
waitforNOTvisible|xpath for cart header

When I import, only the click|id=EE_cart remain in the script.
